quantum/pointing_device_drivers.c

```diff
@@ -165,11 +162,26 @@ const pointing_device_driver_t pointing_device_driver = {
 // clang-format on
 
 #elif defined(POINTING_DEVICE_DRIVER_pimoroni_trackball)
+
+mouse_xy_report_t pimoroni_trackball_adapt_values(clamp_range_t* offset) {
```

Comment:
Was it intended to move this from pimoroni_trackball.c?

Reply:
Yeah. Wanted to move all the hid clamping stuff into the drivers file, as it's more appropriate there, I think.